src/main/java/ui/windows/StatsDisplayData.java

```diff
@@ -78,22 +58,22 @@ public int getGamesWon() {
      * @param gamesWon the number of games won as an integer
      */
     public void setGamesWon(int gamesWon) {
-        this.gamesWon = gamesWon;
+        this.gamesLost = gamesWon;
     }
 
```

Comment:
I think this is also missing a **_get_**GamesWon() method, where you return the gamesWon value.

Reply:
I believe the method above this is the getGamesWon() method?

Reply:
Oh, I missed that, all the get and sets got confused in my head. Sorry!